js/views/pageNavVw.js

```diff
@@ -591,14 +611,175 @@ module.exports = baseVw.extend({
       Backbone.history.navigate(route, {trigger: true});
     });
   },
+  
```

Comment:
Does this have to live in pageNavView? Could we not have a seperate suggestionListView (or something) that encapsulates the functionality to render a list of suggestions with a way to navigate through them? Then pageNavView would be the parent view interfacing with our separate child view.

I think in general we have overly bloated views and it would be more manageable to break isolated functionality into separate views.